src/Types/RedeemerTag.purs

```diff
@@ -37,3 +40,12 @@ instance BoundedEnum RedeemerTag where
   cardinality = genericCardinality
   toEnum = genericToEnum
   fromEnum = genericFromEnum
+
+instance EncodeAeson RedeemerTag where
+  encodeAeson' = case _ of
+    Spend -> encodeTagged' "Spend" {}
+    Mint -> encodeTagged' "Mint" {}
+    Cert -> encodeTagged' "Cert" {}
+    Reward -> encodeTagged' "Reward" {}
+    where
+    encodeTagged' str x = encodeAeson' $ encodeTagged str x (Op encodeAeson)
```

Comment:
Missing a type signature here

Reply:
Thanks, this should have been replaced by the utility I introduced in the `Helpers` module